### ISSUE TITLE
MEC: add yag1, yag2, yag3.

### ISSUE THREAD

**Issue by silkenelson** [pcdshub/Bug-Reports-and-Requests#14]
**Feature Request**
Define aliases.

**Comment by ZLLentz** [comment]
MEC wants the yags to be re-numbered in their python session?

**Comment by silkenelson** [comment]
@slactjohnson feel free too Loe this if it has already been taken care of. Many of these issues come from trying to clear out old Trello cards where it was not very clear what had and had not been taken care of.....

**Comment by egaltier** [comment]
The current commands are used by typing `yag3.insert()` and `yag3.remove()`for example. Is there a way to shorten them to `yag3.in()`and `yag3.out()`for example?

**Comment by ZLLentz** [comment]
Yes, we can make aliases for shorter calls, but we can't use "in" as a function name in Python because it is a reserved keyword. See https://docs.python.org/3/reference/lexical_analysis.html#keywords.

**Comment by egaltier** [comment]
Ah, that's unfortunate... I guess we were using aliases before so that's why it was doable to use `in`. What about `yag3.ins()` then? If the tab completion stops at `in` and wait for the `s` that would be unfortunate as well and we might have to find something different that fits in 3 letters to match `out` :p I might fine tune a bit too much here and that might not be appropriate... !

**Comment by slactjohnson** [comment]
I'm leery of adding aliases to methods. I think we should keep class methods standard everywhere. I don't think that typing yag1.ins<tab>() is much more onerous than typing yag1.in().

**Comment by ZLLentz** [comment]
The implementation would be to drop into this class here:
https://github.com/pcdshub/pcdsdevices/blob/b43da719a36231859abd167a19addee04a8ec5a8/pcdsdevices/inout.py#L96-L114
and simply add
```
ins = insert
out = remove
```
or whatever names we agree on.

If you do have some voodoo for using `in` as a function name/alias I'd be interested to know how, just for curiosity's sake.

**Comment by slactjohnson** [comment]
Interesting, my previous comment did not display as I intended. I meant to say that typing yag1.ins + TAB + () is not much more work than typing yag1.in(). 

I am not a proponent of making the modifications Zach detailed. I think the existing methods are clearly named, and we don't need identical methods cluttering up the tab completion list. I think tab completion is enough to take the sting out of typing a few more characters.

**Comment by silkenelson** [comment]
I agree with Tyler. If we do aliases that chop a few characters of a command, it will clutter up the tab-completion interface. I consider aliases for things where you have to tab complete either 2 or 3 times (in which case we can make them short).
